src/wallet/api.rs

```diff
@@ -1248,7 +1289,7 @@ impl Wallet {
                         amount.to_sat()
                     );
 
-                    Ok(unspents)
+                    Ok(all_utxos)
```

Comment:
This logic can remain as before. We still have the ungrouped `unspents` variable.